src/popup/popupInterfacesSOAP.js

```
@@ -0,0 +1,61 @@
+
+function logDebug(message, ...args) {
+  if (POPUP_SOAP_INTERFACES_DEBUG) {
+    console.log("[POPUP_SOAP_INTERFACES_DEBUG]", message, ...args);
+  }
+}
+
+// use chrome.winfows.getAll to access code in https://devmcnichols.bigmachines.com/rest/v1/interfaceCatalogs/soapCatalog/services/DataTables_v1
+
+chrome.windows.getAll({ populate: true }, function (windows) {
+  for (let i = 0; i < windows.length; i++) {
+    let window = windows[i];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'window' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)